e2e/atlas/clusters_test.go

```diff
@@ -174,7 +186,7 @@ func TestClustersFlags(t *testing.T) {
 			clustersEntity,
 			"update",
 			clusterName,
-			"--diskSizeGB=20",
+			"--diskSizeGB=40",
```

Comment:
[nit] I'd move these sizes to defines like `DefaultSize` `LargeSize` etc so we can reuse easier. Not blocking.

Reply:
to follow colm's suggestion, see https://pkg.go.dev/go.mongodb.org/atlas@v0.10.1/mongodbatlas#pkg-variables